common/logging.go

```diff
@@ -221,7 +221,7 @@ func (l WriterLogger) logToWriter(f io.Writer, prefix string, format string, arg
 }
 
 func logToWriter(f io.Writer, prefix string, format string, args ...interface{}) {
-	_, file, line, ok := runtime.Caller(2)
+	_, file, line, ok := runtime.Caller(3)
```

Comment:
Is this typically wanted?  Is this better if calling from outside the library?  Might be some option that we could add to logging?

Reply:
That's the only way logging works for me. 
e.g. This is a log I am reading right now `[INFO]  text_strata.go:158 scanBand:` as I run a program from https://github.com/unidoc/unipdf-examples.
The file and line number are correct. 
I can't see how 2 would work.